mission_control/navigator_launch/launch/subsystems/lqrrt_node.launch

```diff
@@ -0,0 +1,22 @@
+<launch>
+
+  <node pkg="lqrrt_ros" type="lqrrt_node.py" name="lqrrt_node" output="screen">
+    <!-- Action topic to send commands to -->
+    <param name="move_topic" value="/move_to"/>
+
+    <!-- Topics for odometry and an occupancy grid-->
+    <param name="odom_topic" value="/odom"/>
+    <param name="ogrid_topic" value="/ogrid"/>
+
+    <!-- Pose reference topic - this will be the 'carrot' to follow -->
+    <param name="ref_topic" value="/lqrrt/ref"/>
+
+    <!-- Some visulatzation topics for the path, tree, and other good stuff-->
+    <param name="path_topic" value="/lqrrt/path"/>
+    <param name="tree_topic" value="/lqrrt/tree"/>
+    <param name="goal_topic" value="/lqrrt/goal"/>
+    <param name="focus_topic" value="/lqrrt/focus"/>
+    <param name="effort_topic" value="/lqrrt/effort"/>
```

Comment:
[nit] Unless these are each being used to trigger RVIZ built-in visualizations, consider publishing all these together on a single topic as a single message